CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ cmake_dependent_option(
     "NOT BUILD_SHARED_LIBS" OFF)
 option(BUILD_OBSERVERS "Build performance observers/loggers in caffe2/share/observers directory" OFF)
 option(BUILD_TEST "Build C++ test binaries (need gtest and gbenchmark)" ON)
+option(USE_ARM_COMPUTE "Use ARM Compute Library" OFF)
```

Comment:
The name is ambiguous: users may think they have to enable it for any ARM builds. Please change to `USE_ACL`, `USE_ARMCL`, or `USE_ARM_COMPUTE_LIBRARY`